packages/samples/vuejs/src/assets/styles.css

```diff
@@ -0,0 +1,184 @@
+:root {
```

Comment:
We added layouts recently https://docs.coveo.com/en/atomic/latest/usage/layouts/ which allows us to remove most of the styling here.

Like in https://github.com/coveo/ui-kit/blob/master/packages/atomic/src/pages/index.html

Reply:
Yeah we'll need to do the job and clean up Atomic React and Atomic Angular samples.

Reply:
I'll add a separate Jira ticket for this!